functions/autopost-clear.ts

```diff
@@ -19,7 +19,8 @@ export const autoClearQueue = async (event): Promise<BackgroundProcessResults> =
   }
 
   let errors = false
-  const forceNotify = event.queryStringParameters.force === 'true'
+  const forceClear = event.queryStringParameters.force === 'true'
+  const clearAll = event.queryStringParameters.all === 'true'
```

Comment:
I'm not sure that this clearAll variable accurately represents what we need from this feature.